src/test/credentials/provider/sharedCredentialsProvider.test.ts

```diff
@@ -108,6 +108,16 @@ describe('SharedCredentialsProvider', async () => {
         assertSubstringsInText(sut.validate(), MISSING_PROPERTIES_FRAGMENT, 'aws_secret_access_key')
     })
 
+    it('validation identifies assume role must have source_profile', async () => {
```

Comment:
the wording is hard to parse. suggestion:

```suggestion
    it('"assume role" must have source_profile', async () => {
```

Reply:
I was kinda following the existing pattern - but happy to change ...